tests/drivers/spi/spi_loopback/boards/nrf54lm20dk_nrf54lm20a_cpuapp.overlay

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Test requires following loopback:
+ * P1.13 - P1.14
+ */
+
+&pinctrl {
+	spi21_default: spi21_default {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 1, 24)>,
+				<NRF_PSEL(SPIM_MISO, 1, 13)>,
+				<NRF_PSEL(SPIM_MOSI, 1, 14)>;
+		};
+	};
+
+	spi21_sleep: spi21_sleep {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 1, 24)>,
+				<NRF_PSEL(SPIM_MISO, 1, 13)>,
+				<NRF_PSEL(SPIM_MOSI, 1, 14)>;
+			low-power-enable;
+		};
+	};
+};
+
+&spi21 {
+	status = "okay";
+	pinctrl-0 = <&spi21_default>;
+	pinctrl-1 = <&spi21_sleep>;
+	pinctrl-names = "default", "sleep";
+	overrun-character = <0x00>;
+	zephyr,pm-device-runtime-auto;
+	slow@0 {
```

Comment:
lot of non-compliant dts parts in this PR, please submit a follow up PR fixing all issues introduced in this PR, newline on line 37 and line 42, then fix all instances added in whole PR